doc/ref_model/chapters/chapter07.md

```diff
@@ -154,15 +154,17 @@ Insuring that the security standards and best practices are incorporated into th
 <a name="7.1.1"></a>
 ## 7.4.1 In-scope and Out-of-Scope definition
 
-    *(Declare what should be in scope and what should be out of scope for
-    CNTT security)*
+The scope of the security controls requirements maps to the scope of the CNTT reference architecture.
 
--   Define and note separation of security postures of platform and
-    workload, but that workload is dependent’ upon platform security.
+The CNTT scope is shown below:
 
-    *(Provide some background commentary, from a security perspective,
-    around the inter-relationship between platform and workload security
-    where it is assumed that ‘workload’ leverages ‘platform’ security)*
+![Image of ETSI NFV Mapping](![https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/figures/ch01_etsi_archi_mapping_v2.PNG](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/figures/ch01_etsi_archi_mapping_v2.PNG))
+
+**Figure 7-1**: Mapping to ETSI NFV Architecture
+
+This means that the security of the CNTT requirements must cover the virtual resources (including the virtualisation layer), the hardware resources, and the VIM. It is noted, however that the VIM (Virtualised Infrastructure Manager) is not strictly included into NFVi, but the interactions between the NFVi and VIM are included.
+
+There will be a different set of security requirements for each NFVi reference architecture. In this case, the first reference model is OpenStack.
```

Comment:
the first reference _architecture_ is OpenStack.

Reply:
Thanks, changes made ..